packages/edv-plugin/src/functions/passwordToKey.ts

```diff
@@ -0,0 +1,43 @@
+import crypto from 'isomorphic-webcrypto';
+
+export const passwordToKey = async (
+  password: string,
+  salt: string = 'salt',
+  iterations: number = 100000,
+  digest: string = 'SHA-256'
+): Promise<Uint8Array> => {
+  var saltBuffer = Buffer.from(salt);
+  var passphraseKey = Buffer.from(password);
+
+  return crypto.subtle
+    .importKey('raw', passphraseKey, { name: 'PBKDF2' }, false, [
+      'deriveBits',
+      'deriveKey',
+    ])
+    .then(function (key: any) {
+      return crypto.subtle.deriveKey(
+        {
+          name: 'PBKDF2',
+          salt: saltBuffer,
+          iterations: iterations,
+          hash: digest,
+        },
+        key,
+        // Note: we don't actually need a cipher suite,
+        // but the api requires that it must be specified.
+        // For AES the length required to be 128 or 256 bits (not bytes)
+        { name: 'AES-CBC', length: 256 },
+        // Whether or not the key is extractable (less secure) or not (more secure)
+        // when false, the key can only be passed as a web crypto object, not inspected
```

Comment:
```suggestion
        // When false, the key cannot be inspected; it can only be passed
        // as a web crypto object, no matter whether the key is extractable
        // (less secure) or not (more secure).
```